test/useConfirm.test.js

```diff
@@ -408,7 +408,7 @@ describe("useConfirm", () => {
       expect(deleteCancelled).not.toHaveBeenCalled();
     });
 
-    test("does not close the modal when another component with useConfirm is unmounted", async () => {
+    test("does not close the modal when another component with useConfirm is unmounted and closeOnParentUnmount is true", async () => {
```

Comment:
This test is for when another component with useConfirm unmounts, so in this case the option is not relevant.

```suggestion
    test("does not close the modal when another component with useConfirm is unmounted", async () => {
```